frameworks/digital-outcomes-and-specialists/messages/homepage-sidebar.yml

```diff
@@ -8,16 +8,19 @@ coming:
     You can apply to provide:
 
       - digital outcomes,
-        eg a discovery phase or an online billing application
+        eg a discovery phase for an online billing application
       - digital specialists,
         eg service managers or developers
       - user research studios
       - user research participants
 
-    [Create an account](https://www.digitalmarketplace.service.gov.uk/suppliers/create) to receive an email when Digital Outcomes and Specialists opens.
-
-
-    [Find out how to apply](https://www.digitalmarketplace.service.gov.uk/)
+    [Create an account](https://www.digitalmarketplace.service.gov.uk/suppliers/create) to receive an email when Digital Outcomes and Specialists opens. 
```

Comment:
The change makes sense to me. There’s not much you can do with your account until the framework is open. And there’s no problem with waiting until the framework is open to create an account—you just won’t receive an email.